.github/workflows/prepare-prerelease.yml

```diff
@@ -0,0 +1,64 @@
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+  prepare-release:
+    runs-on: ubuntu-latest
+    if: startsWith(github.event.head_commit.message, '[Release:beta]')
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.BOT_GITHUB_TOKEN }}
+
+      - uses: Alfresco/alfresco-build-tools/.github/actions/load-release-descriptor@v1.15.0
+        id: load-descriptor
+        with:
+          release-descriptor: release.yaml
+
+      - name: Checkout activiti-cloud-application
+        uses: actions/checkout@v3
+        with:
+          repository: 'Activiti/activiti-cloud-application'
```

Comment:
out of curiosity: do we need the `'`?

Reply:
I don't think it's actually needed. I'm going to remove them.